charts/px-backup/templates/NOTES.txt

```diff
@@ -0,0 +1,28 @@
+
+Your Release is named {{ .Release.Name | quote }}
+
+NOTE:
```

Comment:
This needs some headings so that they stand out. Otherwise, it just reads like a blurb of text and uses won't know what to do with it
```suggestion
-----
NOTE:
-----
<text>
----------------
MONITOR INSTALL:
----------------
<text>
```

Reply:
Changed